google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java

```diff
@@ -274,45 +298,52 @@ private static String ensureKmsKeyExistsForTests(
 
   @Test
   public void testClearBucketDefaultKmsKeyName() throws ExecutionException, InterruptedException {
-    String bucketName = dataGeneration.getBucketName();
+    String bucketName = bucketFixture.newBucketName();
     Bucket remoteBucket =
-        storage.create(
-            BucketInfo.newBuilder(bucketName)
-                .setDefaultKmsKeyName(kmsKeyOneResourcePath)
-                .setLocation(KMS_KEY_RING_LOCATION)
-                .build());
+        storageFixtureHttp
```

Comment:
Can we add todo's with the bug number next to these explicit http usages to cleanup and use `storage` once it's available?

Reply:
yeah I can do that, I was kinda just planning to search for storageFixtureHttp and remove the uses one by one but have no real preference.

Reply:
That's probably fine as long as we're tracking it somewhere